app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+
+    if "mv" not in command or len(command.split()) != 3:
```

Comment:
This is not true because when command will equal to hello mv.txt second.txt it will have incorrect behavior

Reply:
done, thnx